src/main/resources/db/migration/V1__Create_Beacons.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS beacons (
+    id SERIAL,
+    beaconType VARCHAR,
+    hexId VARCHAR,
+-- Would it be better to have hexId as Hex type for more efficient queries?
```

Comment:
@zackads raised that postgres has a hex type which would make queries faster for us if we used it 🤔 (I don't fully understand this)

Reply:
We (@matthew-a-carr , @zackads and @Ninamma ) discussed this today, decided to stick with `VARCHAR` for the time being.  Simple to convert a `VARCHAR` with a hext string in it to a `BYTEA` [hex type](https://www.postgresql.org/docs/9.1/datatype-binary.html) in a future migration if necessary.